File: 01-JavaScript/02-change-string-casing.js
const prompt = require("prompt-sync")({ sigint: true });

const word = prompt("Enter any word, all in lowercase: ");
const firstChar = word.slice(0, 1)
const firstUpper = firstChar.toUpperCase();
const restChar = word.slice(1, word.length);
console.log("\nLook, I can turn your word's first character into uppercase: \n" + firstUpper + restChar);

File: 02-Python/02-change-string-casing.py
#!/usr/bin/python

word = input("Enter any word, all in lowercase: \n")

word_first_char = word[0:1]
word_first_char_upper = word_first_char.upper()
word_rest_char = word[1:]

print("\nLook, I can turn your word's first character into uppercase: \n" + word_first_char_upper + word_rest_char)
